dagmc_stats/DagmcStats.py

```diff
@@ -40,3 +40,23 @@ def __init__(self, filename, populate=False):
         self._vol_data = pd.DataFrame(
             columns=['vol_eh', 'surf_per_vol', 'coarseness'])
         self._vol_data = self._vol_data.set_index('vol_eh')
+
+    def get_native_ranges(self, meshset, entity_types):
```

Comment:
let's rework this as a "convenience function" just for the sake of setting this information in the constructor:

```
def __set_native_ranges(self):
```

The meshset is just `self.root_set`. And the entity types are the list you are originally passing in, but we don't need that as a separate passed variable and can have that set in the function directly.

Making it private with the double underscore indicates that this is only to be used as a helper function in the constructor and not as an accessible function to users.